commands/motor/MotorControl.java

```diff
@@ -42,13 +46,39 @@ public MotorControl(Motor motor, Controller controller, int axis, double scale)
 	 * This Command directly controls a Motor based on an axis of the Controller.
 	 * This can allow an Operator to easily control a single Motor from an axis of the Controller.
 	 *
+	 * @param name
+	 * @param motor
+	 * @param controller
+	 * @param axis
+	 */
+	public MotorControl(String name, Motor motor, Supplier<Double> speedSupplier) {
+		this(name, motor, speedSupplier, 1.0, 0.0);
+	}
+
+	/**
+	 * This Command directly controls a Motor based on an axis of the Controller.
+	 * This can allow an Operator to easily control a single Motor from an axis of the Controller.
+	 *
+	 * @param motor
+	 * @param controller
+	 * @param axis
+	 * @param scale
+	 */
+	public MotorControl(Motor motor, Supplier<Double> speedSupplier, double scale) {
```

Comment:
What is the point of this? Where does the scale get factored in/used?

Reply:
Yeah -- it looks like scale is just stored on the instance, and then never used. IMO scale is irrelevant when using speedSupplier; it was used when reading directly from the joystick, but with DoubleSupplier the user could just add the scale factor in the supplier itself.

Nice catch.